addons/l10n_eg_edi_eta/tests/common.py

```diff
@@ -0,0 +1,123 @@
+from datetime import datetime
+
+from pytz import timezone
+
+from odoo.tests import tagged
+
+from odoo.addons.account_edi.tests.common import AccountEdiTestCommon
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestEGEdiCommon(AccountEdiTestCommon):
+    @classmethod
+    def setUpClass(
+        cls, chart_template_ref='l10n_eg.egypt_chart_template_standard', edi_format_ref='l10n_eg_edi_eta.edi_eg_eta'
+    ):
+        super().setUpClass(chart_template_ref=chart_template_ref, edi_format_ref=edi_format_ref)
+
+        cls.frozen_today = datetime(year=2022, month=3, day=15, hour=0, minute=0, second=0, tzinfo=timezone('utc'))
+
+        cls.currency_aed_id = cls.env.ref('base.AED')
+        cls.currency_aed_id.write({'active': True})
+        cls.env['res.currency.rate'].search([]).unlink()
+        cls.env['res.currency.rate'].create(
+            {'currency_id': cls.currency_aed_id.id, 'rate': 0.198117095128, 'name': '2022-03-15'}
+        )
+
+        # Allow to see the full result of AssertionError.
+        cls.maxDiff = None
+
+        cls.company_data['company'].write(
+            {
+                'country_id': cls.env.ref('base.eg').id,
+                'l10n_eg_client_identifier': 'ahuh1pojnbakKK',
+                'l10n_eg_client_secret_1': '1ashiqwhejmasn197',
+                'vat': 'EG1103143170L',
+            }
+        )
```

Comment:
```suggestion
        cls.company_data['company'].write({
                'country_id': cls.env.ref('base.eg').id,
                'l10n_eg_client_identifier': 'ahuh1pojnbakKK',
                'l10n_eg_client_secret_1': '1ashiqwhejmasn197',
                'vat': 'EG1103143170L',
            })
```
As clear in less lines